http/server.go

```diff
@@ -529,6 +530,25 @@ func (h *DrandHandler) Health(w http.ResponseWriter, r *http.Request) {
 	_, _ = w.Write(b)
 }
 
+func (h *DrandHandler) ChainHashes(w http.ResponseWriter, r *http.Request) {
+	chainHashes := make([]string, 0)
+	for chainHash := range h.beacons {
+		if chainHash != common.DefaultChainHash {
+			chainHashes = append(chainHashes, chainHash)
+		}
+	}
+
+	w.Header().Set("Content-Type", "application/json")
+	w.Header().Set("Cache-Control", "no-cache")
```

Comment:
is this really 'no-cache'? this response will not change often. We ideally would like these responses to be handled at the CDN layer most of the time rather than go back to the actual application code layer.
Something like a 5 minute cache for this seems pretty reasonable.

Reply:
Ok